recipes-wpe/wpeframework/include/playerinfo.inc

```diff
@@ -2,11 +2,11 @@
 WPE_PLAYERINFO_AUTOSTART ?= "true"
 WPE_PLAYERINFO_DOLBY_FLAGS = "${@bb.utils.contains('DISTRO_FEATURES', 'dolby', '-DDOLBY_SUPPORT=ON', '-DDOLBY_SUPPORT=OFF', d)}"
 WPE_PLAYERINFO_DOLBY_PLATFORM_IMPL ?= "None"
-DEPENDS_append_rpi = " gstreamer1.0"
+WPE_PLAYERINFO_DEP ?= ""
```

Comment:
I would use even weaker assignment( ??=). Let's keep that in mind when we will be adding new configuration options

Reply:
done